Translations-Recommendations-Guide.md

```diff
@@ -0,0 +1,99 @@
+# These are some recommendations for translating FreeCodeCamp to any language
+
+Vladimir Támara Patiño (vtamara). [vtamara@pasosdeJesus.org](mailto:vtamara@pasosdeJesus.org) Public Domain according to the Colombian law.
```

Comment:
With the comment of the end with link to the original, you can remove my name. I guess fine for @justinian336 (?)